### PR TITLE
Create 02227-KimayaUrdhwareshe.py

### PR DESCRIPTION
**Pull request by wonderwoman23**
Submission for task 2.

### CODE REVIEW

**Review by melissaugrai — CHANGES_REQUESTED** [review]
I believe you need to save this file .py,  also missing your email at the end after the word task

File: scripts/02227-KimayaUrdhwareshe [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Name = "Kimaya Urdhwareshe"
+ID = "HNG-02227"
+Language = "Python"
+
+
+print("Hello World, this is {0} with HNGi7 ID {1} using {2} for stage 2 task.".format(Name, ID, Language))
```
Comment (tosinibrahim96) [inline scripts/02227-KimayaUrdhwareshe]: Please add your email address to this out. Here's a sample format `Hello World, this is Ciroma Adekunle with HNGi7 ID HNG-1010 using Javascript for stage 2 task abc@example.com`

File: scripts/02227-KimayaUrdhwareshe [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Name = "Kimaya Urdhwareshe"
+ID = "HNG-02227"
+Language = "Python"
+
```
Comment (wonderwoman23) [inline scripts/02227-KimayaUrdhwareshe:4]: Email = "kimaya23@gmail.com"

**Comment by wonderwoman23** [comment]
Done! Thank you.

**Review by melissaugrai — APPROVED** [review]

**Review by ifatoki — APPROVED** [review]

**Pull request merged**